sdk/monitor/monitor-opentelemetry-exporter/src/export/metric.ts

```diff
@@ -33,9 +34,11 @@ export class AzureMonitorMetricExporter
    * Initializes a new instance of the AzureMonitorMetricExporter class.
    * @param AzureExporterConfig - Exporter configuration.
    */
-  constructor(options: AzureMonitorExporterOptions = {}) {
-    super(options);
+
+  constructor(options: AzureMonitorExporterOptions = {}, isStatsbeat?: boolean) {
```

Comment:
I'm a little worried about having isStatsbeat as part of the constructor here, it will cause user confusion, having a method  @internal public _setIsStatsbeat method or something like that would be better so customers will not see this immediately when using the Metrics exporter, that will require to have all Statsbeat initialization as part of that instead of having in the exporter constructor, let me know what you thing about this one

Reply:
I definitely agree that this could be confusing to customers. Just to clarify what you're thinking, it'd be a public method inside the `AzureMonitorMetricExporter` called from the `statsbeatMetrics` file that sets a flag to indicate that we'll be exporting statsbeat envelopes?

Reply:
Created a new `AzureMonitorStatsbeatExporter` to handle exporting Statsbeat that is separate from the `AzureMonitorMetricExporter` to clean up its constructor.